.cirrus.yml

```diff
@@ -227,6 +227,7 @@ codecov_task:
     pip install pytest-cov codecov
 
   check_requirements_script:
+    check-manifest
```

Comment:
This should fix the failure, based on the above syntax:

```suggestion
  check_requirements_script: |
    check-manifest
```

Reply:
thanks that fixed it